app/javascript/src/assets/scss/widgets/_sidemenu.scss

```diff
@@ -1,120 +1,128 @@
 .side-menu {
   i {
-      min-width: 2rem;
+    min-width: 2rem;
   }
 }
 
-
 .sidebar {
-  width: $nav-bar-width;
-  z-index: 1024 - 1;
-  overflow-x: hidden;
   @include border-normal-right;
   @include background-white;
   @include full-height;
   @include margin(0);
   @include space-between-column;
+  overflow-x: hidden;
+  width: $nav-bar-width;
+  z-index: 1024 - 1;
 
   //logo
   .logo {
     img {
+      @include padding($woot-logo-padding);
       // width: $woot-logo-width;
       // height: $woot-logo-height;
-      @include padding($woot-logo-padding);
     }
   }
+}
 
-  .main-nav {
-    @include flex-weight(1);
-    padding: 0 $space-medium - $space-one;
-    @include scroll-on-hover;
+// bottom-nav
+.bottom-nav {
+  @include flex;
+  @include space-between-column;
+  @include padding($space-one $space-normal $space-one $space-one);
+  @include flex-direction(column);
+  @include border-normal-top;
+  position: relative;
 
-    a {
-      color: $color-gray;
-      font-size: $font-size-default;
-      border-radius: $space-smaller;
-      &:before {
-        margin-right: $space-slab;
-      }
+  .dropdown-pane {
+    @include elegant-card;
+    @include border-light;
+    left: 18%;
+    top: -160%;
+    visibility: visible;
+    width: 80%;
+    z-index: 999;
+
+    &::before {
+      @include arrow(bottom, $color-white, $space-slab);
+      bottom: -$space-slab;
+      position: absolute;
+      right: $space-slab;
     }
+  }
 
-    .menu-title {
-      font-size: $font-size-medium;
-      color: $color-gray;
-      margin-top: $space-medium;
-      > span {
-        margin-left: auto;
-      }
-      & + ul > li > a{
-        @include padding($space-micro null);
-        line-height: $global-lineheight;
-        color: $medium-gray;
-      }
+  .active {
+    border-bottom: 2px solid $medium-gray;
+  }
+}
+
+.main-nav {
+  @include flex-weight(1);
+  @include scroll-on-hover;
+  padding: 0 $space-medium - $space-one;
+
+  a {
+    border-radius: $space-smaller;
+    color: $color-gray;
```

Comment:
This got overridden by foundation classes, could you please check?

<img width="228" alt="Screenshot 2019-10-05 16 56 15" src="https://user-images.githubusercontent.com/2246121/66252694-0e6fe880-e791-11e9-966d-043f0b125058.png">

Reply:
Fix it in new correct PR: https://github.com/chatwoot/chatwoot/pull/122